wgpu-hal/src/vulkan/command.rs

```diff
@@ -503,6 +503,8 @@ impl crate::CommandEncoder for super::CommandEncoder {
                     for triangles in in_geometries {
                         let mut triangle_data =
                             vk::AccelerationStructureGeometryTrianglesDataKHR::default()
+                                // by default this is IndexType::UINT16 because IndexType::NONE_KHR is only allowed here
```

Comment:
I don't really understand what this comment is saying, can you elaborate?

Reply:
None_KHR is only provided by extension acceleration structures (for use while building) so it is by not the default, since it is not the default, but here if we don't have an index buffer we need to have None_KHR, so we need to set it here.

I've realized that the comment is confusing, I'll look at updating it